workflow/workflow.go

```diff
@@ -166,9 +166,10 @@ func (c *Cache) Refresh() (files []string, err error) {
 	}
 
 	// remove deleted workflows
+	missing := make([]string, 0)
 	for key := range c.Workflows {
 		found := false
-		for _, v := range files {
+		for _, v := range sts {
```

Comment:
This is the fix. 
sts is a list of all files in the path
files is a list of all files that changed